notebooks/demo2/transformer-relevance-sparse.ipynb

```diff
@@ -0,0 +1,2042 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #40.        trainer.evaluate(eval_dataset=encoded_climate_dataset['test'])
</pre><p>we could add output of this execution in the metrics as well. This basically gives you more runtime info for the test set.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/os-climate/aicoe-osc-demo/pull/234/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>